### ISSUE TITLE
Request: Get tests running or delete them

### ISSUE THREAD

**Issue by paulrd** [yogthos/memory-hole#37]
It seems that development has broken tests some time ago. Should we delete them for now?

**Comment by yogthos** [comment]
yeah the tests are just the default generated stubs, so would make sense to delete the failing ones
